frontend/frontend-filters-search.php

```diff
@@ -159,12 +160,18 @@ public function wp_print_footer_scripts() {
 					var ih = document.createElement( 'input' );
 					ih.type = 'hidden';
 					ih.name = 'lang';
-					ih.value = '$lang';
+					ih.value = '{$lang}';
 					e[i].parentNode.appendChild( ih );
 				}
 			}
+		}";
+
+		$type_attr = current_theme_supports( 'html5', 'script' ) ? '' : ' type="text/javascript"';
+
+		if ( $type_attr ) {
+			$js = "/* <![CDATA[ */\n{$js}\n/* ]]> */";
 		}
-		//]]>";
-		echo '<script type="text/javascript">' . $js . '</script>'; // phpcs:ignore WordPress.Security.EscapeOutput
+
+		echo "<script{$type_attr}>\n{$js}\n</script>\n"; // phpcs:ignore WordPress.Security.EscapeOutput
```

Comment:
Same as in Polylang Pro https://github.com/polylang/polylang-pro/pull/1779#discussion_r1301437991
Couldn't we use printf instead?